modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/dump/IgniteCacheDumpSelf2Test.java

```diff
@@ -330,7 +330,8 @@ public void testCustomLocation() throws Exception {
                 null,
                 false,
                 false,
-                true
+                true,
```

Comment:
Let's add in this class tests:
1. compressed dump created files with ".zip" suffix
2. comressed dump is less than non-compressed dump